cve_bin_tool/checkers/openswan.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for openswan
+
+https://www.cvedetails.com/product/57217/Xelerance-Openswan.html?vendor_id=20146
+
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """returns version information for openswan as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be openswan if openswan is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of openswan or contains one
+    version gives the actual version number
+
+    VPkg: xelerance, openswan
+    """
+    regex = [r"Openswan ([0-9]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+
+    binary_names = [
+        "klipsdebug",
+        "_copyright",
+        "showhostkey",
+        "ranbits",
+        "eroute",
+        "showpolicy",
+        "spi",
```

Comment:
Yes. And since we have a lot of binaries which contain the strings we are looking for, I guess it will be better if we do not include spi in binary files list. What do you think?

Reply:
I could go either way.  If it's got the version string, it's going to be detected in the "contains" check either way, so probably the least noise option is to leave it out of the file checks so we don't get the wrong file turning up with version UNKNOWN?

Reply:
Yes, you are right. Better we leave it out.